src/shared/hooks/utils/usePaginationAtomHook.tsx

```diff
@@ -47,7 +47,7 @@ export const usePaginationAtomHook = <TEntity, TTransformed = TEntity>({
   const handleDataUpdate = (data: TEntity[]) => {
     if (data) {
       // TODO: This is a temporary workaround - Replace with count
-      if (data.length < itemLimit - 1) {
+      if (data.length < itemLimit - 2) {
```

Comment:
are we receiving items count less by 2 ?

Reply:
Yeah that's why the scroll on the global feed was not working